bitnami/metallb/templates/controller/deployment.yaml

```diff
@@ -91,6 +94,13 @@ spec:
             - --port={{ .Values.controller.containerPorts.metrics }}
             - --cert-service-name=metallb-webhook-service
             - --log-level={{ .Values.controller.logLevel }}
+            - --webhook-mode={{ .Values.controller.webhookMode }}
+            {{- if .Values.controller.tlsMinVersion }}
+            - --tls-min-version={{ .Values.controller.tlsMinVersion }}
+            {{- end }}
+            {{- if .Values.controller.tlsCipherSuites }}
+            - --tls-cipher-suites={{ .Values.controller.tlsCipherSuites }}
+            {{- end }}
```

Comment:
In other apps we have the extraArgs value to allow extra settings added in the future